main/src/com/google/refine/importing/DefaultImportingController.java

```diff
@@ -290,7 +293,15 @@ static public void writeErrors(JsonGenerator writer, List<Exception> exceptions)
             e.printStackTrace(new PrintWriter(sw));
             
             writer.writeStartObject();
-            writer.writeStringField("message", e.getLocalizedMessage());
+            String message = e.getLocalizedMessage();
+            
+            if(e.getCause() instanceof JsonParseException) {
```

Comment:
This code is specific to the JSON importer, but it was added to a general purpose importing class. This is a problem: writing importers should not require making changes to generic classes like this.

Instead, we should have a generic way to report parsing exceptions to the user. So I propose we do this:
* Catch the `JsonParseException` in the JSON importer;
* Extract the message as you are doing here;
* Store this message in a new exception, that we could introduce in this PR but could then be reused by other importers (we could call it `ImportingParserException`);
* Then we can still have your `isImportingParserException` logic with `"core-views/check-format"` as message that we concatenate.